p4include/pna.p4

```diff
@@ -474,6 +474,70 @@ extern Digest<T> {
 }
 // END:Digest_extern
 
+struct mod_mirror_profile_cfg_t {
```

Comment:
This is MEV specific
There is no structure needed in pna.p4
This structure could even be a user defined structure